Library/Formula/anttweakbar.rb

```diff
@@ -64,3 +63,16 @@ def install
          }
          if (gl_dyld)
          {
+diff --git a/src/Makefile.osx b/src/Makefile.osx
```

Comment:
Probably create a ticket in their issue tracker, describe the problem, describe how this fixes the problem, link to the ticket as a comment above with an explanation of why Homebrew needs it. Thanks!